requirements.in

```diff
@@ -0,0 +1,4 @@
+ansible
+docker<3.0.0
```

Comment:
Nice touch here, and thanks for linking to corresponding upstream issue in the commit message. :+1: Good ol' @geerlingguy, owe that person some :beers:.